integration-cli/docker_cli_daemon_test.go

```diff
@@ -466,17 +466,6 @@ func (s *DockerSuite) TestDaemonStartWithDaemonCommand(c *check.C) {
 		d.Stop()
 	}
 
-	// `docker -l info daemon --selinux-enabled`
-	// should error out
-	for _, f := range flags {
-		d := NewDaemon(c)
-		d.GlobalFlags = f[common]
-		if err := d.Start(f[daemon]...); err == nil {
-			d.Stop()
-			c.Fatalf("Daemon should have failed to start with docker %v daemon %v", d.GlobalFlags, f[daemon])
-		}
-	}
-
```

Comment:
these tests pass `daemon` flags as global flags,  duplicated with the following tests.